tests/TestDeliveryFeatureRouteKeys.py

```
@@ -0,0 +1,64 @@
+        """初始化阶段 task.config 为 None 时不崩溃，仅注册 default_config。"""
+        task = SimpleNamespace(default_config={}, config=None)
+        feature = self.make_feature(task)
+        feature._register_route_keys()
+        self.assertIn(self.ROUTE_KEY, task.default_config)
+        self.assertIn(self.END_KEY, task.default_config)
+        self.assertIn(DeliveryTask.CFG_SCROLL_ENABLE, task.default_config)
+        self.assertIs(task.default_config[DeliveryTask.CFG_SCROLL_ENABLE], False)
+        self.assertEqual(task.default_config[self.ROUTE_KEY], "")
+
+    def test_register_route_keys_fills_runtime_config(self):
+        """运行时 config 已就绪时补齐路线键与滚动开关，且不覆盖用户已有键。"""
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)